samples/confluent-schema-registry-integration/pom.xml

```diff
@@ -1,12 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	<groupId>org.springframework.cloud</groupId>
 	<artifactId>confluent-schema-registry-integration</artifactId>
 	<version>4.0.0-SNAPSHOT</version>
 	<packaging>pom</packaging>
 	<name>confluent-schema-registry-integration</name>
-	<description>confluent-schema-registry-integration</description>
+	<description>Confluent Schema Registry Integration Sample App</description>
```

Comment:
Should we try to put somethng descriptive here for all of the samples or simply omit the element altogether if we just end up copyinhg the "name" value into this?

Reply:
Descriptive is better, IMO.

Reply:
Sounds good. Lets keep an eye out on each one as we migrate over and try to make sure we give them something descriptive.